app/helpers/google_analytics_helper.rb

```diff
@@ -0,0 +1,9 @@
+module GoogleAnalyticsHelper
+  def google_analytics_enabled?
+    ENV.has_key?('GA_TRACKING_ID')
```

Comment:
Would be tempted to have this as `ENV['GA_TRACKING_ID'].present?` - on the off chance some changes a config in the future by setting the env var to blank instead of actually removing it from an App Services config

Reply:
I've been thinking on this, its easy to set an env var to blank and probably as likely as one removed so lets get something checking this isn't blank as well